### ISSUE TITLE
Improve, expand, and document keyboard shortcuts

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#1202]
Currently, users can annotate selected text by typing `a` to create an annotation or `h` to create a highlight. This meets WCAG guidelines but could use some improvement and new additions. For example:

- Enabling/disabling the browser extension
- Opening and closing the sidebar
- Moving from document to sidebar & vice versa
- Interacting with an annotation: Editing, Replying, Flagging, and Deleting
- Editing an annotation: toggle between preview and edit mode

Notes
- All of these commands should have some kind of modifier (CTRL or CMD). We should also ensure that these do not interfere with commonly used keyboard shortcuts in the browser or other assistive technology. 
- Keyboard shortcuts should be documented in the KB

To Do:

- [ ] Make a list of "actions" someone would need to do within the app
- [ ] Propose a set of keyboard shortcuts for each of these options (might be helpful to research commonly used shortcuts to avoid)
- [ ] Meet with our contacts at Benetech to validate those suggestions
- [ ] Get this issue on our product backlog to be prioritized among our other work

**Comment by mattdricker** [comment]
Example of potential for conflict between shortcut keys: [Using keyboard shortcut 'h' with PDF changes pointer to Hand Tool](https://github.com/hypothesis/support/issues/201)

**Comment by mattdricker** [comment]
**Proposed list of actions to trigger with a keyboard shortcut:**

- Activate Hypothesis extension
- Open/close sidebar
- Navigate to next/previous annotation card
- Navigate to next/previous selected document text
- Search

In Editor:
- Toggle between write & preview
- Post to group
- Post to Only Me
- Cancel

[DONE]
* ~~Annotate~~ `a`
* ~~Highlight~~ `h`

**Comment by mattdricker** [comment]
Some additional notes:

> - Navigate to next/previous annotation card
> - Navigate to next/previous selected document text

It may make sense for these to use same KB shortcut, with the difference being whether sidebar is open or shut.

Also, if a user has navigated to a selected text in the document, opening the sidebar should transfer that focus to the related annotation card. Vice versa with an annotation card selected when the sidebar is closed.

**Comment by jeremydean** [comment]
Adding a detail I noted during the OSU call.

The "h" shortcut for "Highlight" conflicts with the common use of "h" as a shortcut to navigate "Headers."

While there is a "pass through mode" in screen readers to avoid this conflict, not all screen reader users will be that sophisticated. 

@mattdricker you had mentioned something about work we had planned on "modifier keys" that might address this.

**Comment by mattdricker** [comment]
@jeremydean yes, we will plan on using modifier keys (e.g. ctrl + alt) with all of our kb shortcuts to avoid these conflicts. Modifier keys currently TBA.

**Comment by mkdir-washington-edu** [comment]
Discussion on this topic will likely solve https://github.com/hypothesis/support/issues/201.

**Comment by chrisshaw** [comment]
**Issue was linked to Harvestr Discovery:**  [Use only my keyboard](https://app.harvestr.io/components/0/list/V0ecSWmJl)

**Comment by tripluoffthoranlz** [comment]
> Some additional notes:
> 
> > * Navigate to next/previous annotation card
> > * Navigate to next/previous selected document text
> 
> It may make sense for these to use same KB shortcut, with the difference being whether sidebar is open or shut.
> 
> Also, if a user has navigated to a selected text in the document, opening the sidebar should transfer that focus to the related annotation card. Vice versa with an annotation card selected when the sidebar is closed.

Hi @jeremydean ,

Do you know if they've gotten anywhere on "Navigate to next/previous annotation card?"
